tests/test_cli/test_cfg_subcmd.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import os
+import shutil
+
+from test_cli import capture
+this_folder = os.path.dirname(__file__)
+
+def test_cfg_help(python_version):
+    """Test if Manim successfully adds configparsers when a subcommand is invoked."""
+    command = f"cd {this_folder} && {python_version} -m manim cfg --help"
+    out, err, exitcode = capture(command, use_shell=True)
+    assert exitcode == 0, f"The cfg subcommand is not working as intended."
+
+def test_cfg_show(python_version):
+    """Test if the `manim cfg show` command works as intended."""
+    command = f"cd {this_folder} && {python_version} -m manim cfg show"
+    out, err, exitcode = capture(command, use_shell=True)
+    assert exitcode == 0
+    assert f"{os.path.sep}tests{os.path.sep}".encode("utf-8") in out, err
+
+def test_cfg_export(python_version):
+    """Test if the `manim cfg export` command works as intended."""
+    command = f"cd {this_folder} && {python_version} -m manim cfg export --dir temp"
+    out, err, exitcode = capture(command, use_shell=True)
+    assert exitcode == 0
```

Comment:
Same comment as above :
```suggestion
    assert exitcode == 0, f"The cfg subcommand export is not working as intended.\nError : {err}"
```